literature.bib

```diff
@@ -74,6 +74,44 @@ @inproceedings{KHS+:VaMoS24
 	note = {To appear}
 }
 
+
+@mastersthesis{Raab23,
+	author = {Raab, Heiko},
+	title = {{Incrementally Adapting d-DNNFs to Cope with Feature-Model Evolution}},
+	school = UniUlm,
+	address = {Germany},
+	type = {Master's Thesis},
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting},
+	month = NOV,
+	year = 2023,
+	note = {To appear}
+}
+
+@mastersthesis{Loth23,
+	author = {Loth, Jacob},
+	title = {{Projected d-DNNF Compilation for Feature Models}},
+	school = UniUlm,
+	address = {Germany},
+	type = {Master's Thesis},
+	typo3Tags = {SupervisorCS, SupervisorTT, FMCounting},
+	month = NOV,
+	year = 2023,
+	note = {To appear}
+}
+
+@mastersthesis{Dunkel23,
+	author = {Dunkel, Raphael},
+	title = {{One Solver to Rule All Feature Models — Or Not? Addressing the Algorithm Selection Problem for \#SAT}},
```

Comment:
Replace `—` by `--` because some programs complain. Maybe check via our validate.py

Reply:
Done, thanks for the quick feedback!